src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+
+import java.util.Scanner;
+
 public class Duke {
+
+
     public static void main(String[] args) {
+        String divider = "\t____________________________________________________________";
+        Scanner in = new Scanner(System.in);
+       // ArrayList<Tasks> taskList = new ArrayList<>();
+        String line;
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.print(divider + '\n'
+                + " \t Hello! I'm Duke\n\t What can I do for you?\n" + divider + "\n\n");
+        line = in.nextLine(); //first input
+        while (!line.equals("bye")) {
+            String[] userInput = line.split(" ", 2);
+            String command = userInput[0];
```

Comment:
Good logic, but the code can be further abstracted into different classes, for example, a command manager class to manage commands instead of putting everything in main:D